### DISCUSSION TITLE
run on 3rd party drones

### DISCUSSION THREAD

**Discussion by uzzy-buzzy** [jyjblrd/Mocap-Drones#60] [Q&A]
hey, as you know, most of the laymen who watched your video would probably not build the whole drone themselves. Rather they ll probably use a cheap third party chinese drone to get the job done. So i m cuious as to:

A) why didnt u retrofit some cheap chinese drone, it ld be less crappy and have a lot more power

B) what whould it take to make this work on some third party drone? would adding some infrared L.E.Ds and soddering the microcontroller to the drone s remote (to issue commands through the remote) be enough? or do we need to do more?

**Comment by jyjblrd** [comment]
I did consider modifying a cheap Aliexpress drone, however I realised it would be more trouble than it was worth. 
- They can only be controlled by the crappy controller that comes with the drone, as opposed to my drone which can be controlled by proper full sized controllers
- There is no way to tune the PID values, etc
- The battery life is a lot worse, ~2 minutes compared to my drones ~10 minutes. This is also the reason why my drone has less power, as it has to carry a large battery

Generally, this was also a very complex project and building my own drone allowed me to have full control over the drone's firmware and hardware. Using a prebuilt cheap drone would have required me to do reverse engineering and may have had reliability issues. I can't imagine that it would be very fun to try connect the esp32 up to the remote controller to control the drone.

Hope that makes sense.
